<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>19</y>
     <width>691</width>
     <height>291</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>691</width>
     <height>291</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="ST_SRV">
      <property name="minimumSize">
       <size>
        <width>461</width>
        <height>53</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>461</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="ST_QL">
      <property name="minimumSize">
       <size>
        <width>469</width>
        <height>53</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>469</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="ST_QP">
      <property name="minimumSize">
       <size>
        <width>469</width>
        <height>53</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>469</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="ST_QS">
      <property name="minimumSize">
       <size>
        <width>469</width>
        <height>53</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>469</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="ST_QD">
      <property name="minimumSize">
       <size>
        <width>469</width>
        <height>53</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>469</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
